core/lib/dal/src/models/storage_block.rs

```diff
@@ -43,6 +43,8 @@ pub struct StorageL1BatchHeader {
     pub bootloader_code_hash: Option<Vec<u8>>,
     pub default_aa_code_hash: Option<Vec<u8>>,
     pub protocol_version: Option<i32>,
+    pub system_logs: Vec<Vec<u8>>,
+    pub compressed_state_diffs: Option<Vec<u8>>,
```

Comment:
mention that it might not be set for older batches, right? 
(are there any other reasons why this might be missing?)

Reply:
nope, every batch will have 7 or 8 system logs (the 8th being present if there is a network update)